src/chunkserver/datastore/chunkserver_chunkfile.cpp

```diff
@@ -188,8 +187,10 @@ CSErrorCode CSChunkFile::Open(bool createFile) {
         && metaPage_.sn > 0) {
         char buf[pageSize_];  // NOLINT
         memset(buf, 0, sizeof(buf));
+        metaPage_.version = FORMAT_VERSION_V2;
```

Comment:
> Change this version in MetaPage's constructor is better, but some unit tests may be affected.

yeap, it's a better implementation.

Reply:
> Change this version in MetaPage's constructor is better, but some unit tests may be affected.

When chunkserver start, it will scan all chunk file in the copyset directory, and these chunk file is not clean. so if we change the version in MetaPage's constructor, these dirty chunk file will cause confuse.